testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
+
+function testInChromeOnDesktop(name, testCallback) {
+    if(devices.real().deviceType === "desktop" && browser.webkit) {
+        QUnit.testInActiveWindow.call(null, name, testCallback);
+    } else {
+        QUnit.skip.call(null, name, testCallback);
+    }
+}
+
+function checkTextOverflow(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.textOverflow, expectedValue, "textOverflow");
+}
+
+function checkWhiteSpace(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element);
+    assert.equal(style.whiteSpace, expectedValue, "whiteSpace");
+}
+
+function checkOverflowX(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
```

Comment:
You can extract `checkStyle` method with `checkParents` param for the background color style

Reply:
I have changed code and now the only 'element' argument is required:
function getOverflowX(element) {
    return window.getComputedStyle(element).overflowX;
}

In tests, it used in this way:
    assert.strictEqual(CheckStyleHelper.getOverflowX($template[0].parentNode), "hidden", "overflowX");

Currently our code declares the 'hidden' value for the 'overflowX' CSS property so content of my template is hidden. Anyone can expect that this behavior will not be changed in future (w/o bc).

Whether or not this code splitting is better?